### PR TITLE
Added Equilibrium Test Network logos & endpoints

### PR DESCRIPTION
**Pull request by mikolajsobolewski**
Added endpoints, logos and spec of Equilibrium Test Network
https://equilibrium.io/en
https://testnet.equilibrium.io/

### CODE REVIEW

File: packages/apps-config/src/api/spec/equilibrium.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  BalancesAggregate: {
+    total_issuance: 'Balance',
+    total_debt: 'Balance'
+  },
+  VestingInfo: {
+    locked: 'Balance',
+    perBlock: 'Balance',
+    startingBlock: 'BlockNumber'
+  },
+  LookupSource: 'AccountId',
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/equilibrium.ts:53]: Which version of Substrate is it using - if pre-2.0 it needs to have `RefCount: 'u8'` - 2.0 (which the API supports) changed the RefCount type, which affects balance lookups.

File: packages/apps-config/src/settings/endpoints.ts [hunk 2]
```diff
@@ -247,6 +247,11 @@ function createTestNetworks (t: TFunction): LinkOption[] {
       info: 'westend',
       text: t<string>('rpc.westend', 'Westend (Polkadot Testnet, hosted by Parity)', { ns: 'apps-config' }),
       value: 'wss://westend-rpc.polkadot.io'
+    },
+    {
```
Comment (jacogr) [inline packages/apps-config/src/settings/endpoints.ts:251]: Move this up to be alphabetical.

File: packages/apps-config/src/ui/logos/index.ts [hunk 3]
```diff
@@ -34,6 +34,7 @@ import nodeRobonomics from './nodes/robonomics.svg';
 import nodeStafi from './nodes/stafi.png';
 import nodeSubsocial from './nodes/subsocial.svg';
 import nodeSubstrate from './nodes/substrate-hexagon.svg';
+import nodeEquilibrium from './nodes/equilibrium.svg';
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:37]: Move the import and subsequent use up to be alphabetical (makes maintenance easier)

**Review by jacogr — COMMENTED** [review]
Some small comments and a question on `RefCount` (changed in the Substrate 2.0 release)

File: packages/apps-config/src/settings/endpoints.ts (hunk 2 above)
Comment (mikolajsobolewski) [reply packages/apps-config/src/settings/endpoints.ts]: I added equilibrium below the dusty. it is ok?

**Comment by jacogr** [comment]
Ok, will merge as-is. If the balances do not show, it is due to RefCount being wrong.

**Review by jacogr — APPROVED** [review]
Thank you!

**Pull request merged**